### PR TITLE
python3Packages.skidl: unstable-2020-09-15 -> 1.0.0

### PR DESCRIPTION
**Pull request by dotlambda**
###### Motivation for this change
don't use unreleased versions

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

I did not test whether this actually works.
@matthuszagh Can you test it?

### CODE REVIEW

File: pkgs/development/python-modules/skidl/default.nix [hunk 1]
```diff
@@ -1,45 +1,37 @@
...
+    rev = version;
+    sha256 = "1m0hllvmr5nkl4zy8yyzfgw9zmbrrzd5pw87ahd2mq68fjpcaqq5";
   };
 
   propagatedBuildInputs = [
-    requests
     future
     kinparse
-    enum34
     pyspice
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/skidl/default.nix:24]: Not sure about this one https://github.com/xesscorp/skidl/blob/1.0.0/setup.py#L33

Comment (dotlambda) [reply pkgs/development/python-modules/skidl/default.nix:24]: It's used in e.g. https://github.com/xesscorp/skidl/blob/7c7e07e4d48f8f629dd6e66c94035de86bae67fe/skidl/tools/spice.py so I left it in.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 128658` at 5b85a6b1 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.skidl</li>
    <li>python39Packages.skidl</li>
  </ul>
</details>

**Comment by risicle** [comment]
`nixpkgs-review` happy, macos 10.15

**Pull request merged**